spec/components/add_another_spec.rb

```diff
@@ -45,4 +45,16 @@ def default_items
 
     assert_select ".js-add-another__empty div.empty"
   end
+
+  it "adds a data-empty-fields attribute of 'false' when `empty_fields` is not set" do
+    render_component({ items: default_items })
+
+    assert_select "div[data-module='add-another'][data-empty-fields='false']"
+  end
+
+  it "adds a data-empty-fields attribute of 'true' when `empty_fields` is set to 'true'" do
+    render_component({ items: default_items, empty_fields: true })
+
+    assert_select "div[data-module='add-another'][data-empty-fields='true']"
+  end
```

Comment:
Does the behaviour need to be tested e.g. it renders the empty state correctly?

Reply:
Good point: I've updated these tests a bit to provide duplicate and change them for the "Default" and "Start empty" options, which is probably the clearest way to present what it being tested.